R/model.R

```diff
@@ -60,15 +60,22 @@ enw_obs_as_data_list <- function(pobs) {
   )
   latest_matrix <- as.matrix(latest_matrix[, -1])
 
+  # get new confirm for processing
+  new_confirm <- data.table::copy(pobs$new_confirm[[1]])
+  data.table::setorderv(new_confirm, c("reference_date", "group",  "delay"))
+
+  # get flat observations
+  flat_obs <- new_confirm$new_confirm
```

Comment:
Is there any risk regarding sorting order?

Reply:
So I put the sorting here as yes if the data is resorted after preprocessing but before any modelling there will be some issues. I think having sorting here though mitigates this risk though there could be some edge risk with the other datasets. Probably worth hardening at a later date.